rootston/seat.c

```diff
@@ -819,6 +1158,13 @@ void roots_seat_set_focus(struct roots_seat *seat, struct roots_view *view) {
 		wlr_seat_keyboard_notify_enter(seat->seat, view->wlr_surface,
 			keyboard->keycodes, keyboard->num_keycodes,
 			&keyboard->modifiers);
+		/* FIXME: Move this to a better place */
+		struct roots_tablet_pad *pad;
```

Comment:
Yeah, this definitely shouldn't be here. Probably near `wlr_pointer_send_enter`.

Reply:
Those are tablet pads that are set here, they are closer to keyboards than pointers (tools are more pointer-ish).

In theory, this should probably be decoupled from the seat, but this PR is getting large enough, which is why I didn't do that for now and kept this working situation.

Reply:
Oh, right. Yeah, that's fine then.